examples/common/pigweed/protos/fabric_bridge_service.proto

```diff
@@ -7,6 +7,17 @@ package chip.rpc;
 // Define the message for a synchronized end device with necessary fields
 message SynchronizedDevice {
   uint64 node_id = 1;
+
+  string unique_id = 2;
+  string vendor_name = 3;
+  uint32 vendor_id = 4;
+  string product_name = 5;
+  uint32 product_id = 6;
+  string node_label = 7;
+  uint32 hardware_version = 8;
+  string hardware_version_string = 9;
+  uint32 software_version = 10;
+  string software_version_string = 11;
```

Comment:
Would you also be able to plumb in if the device is ans ICD device? It is okay if you cannot, but I was hoping that ultimately whatever is calling AddSynchronizedDevice would also provide this

Reply:
I would like to do that in some ICD-specific followups as the code paths are likely somewhat different (I think ICD seems to affect feature map, which in turn makes a command available that is not there otherwise)